Sources/Concrete/Controller/SpeechController.swift

```diff
@@ -0,0 +1,126 @@
+//
```

Comment:
Do you think this class could use UTs or would this be overwork for not much benefits? 
In that case you can inject the AVAudioEngine and mock it instead of how you're initialising it now.

Reply:
There's no real unit testing you can do for this class. It should mostly be functional

Reply:
Yes agreed after reading everything